ManageWikiExtensions.php

```diff
@@ -1701,6 +1701,17 @@
 		],
 		'section' => 'specialpages',
 	],
+	'semanticdrilldown' => [
+        	'name' => 'Semantic Drilldown',
```

Comment:
```suggestion
        	'name' => 'SemanticDrilldown',
```
Needs to match name in extension.json